### DISCUSSION TITLE
PR의 필수 review로 설정되어 있는 인원보다 활동 인원이 부족합니다.

### DISCUSSION THREAD

**Discussion by getCurrentThread** [BaekjoonHub/BaekjoonHub#35] [General]
2명의 approve review가 필요하지만, PR 작성자는 approve review 작성이 불가능하기에 현실적으로 처리가 어렵습니다.
활동 가능한 인원이 3명 이상 시점에서 적용하는 것이 적절할 것으로 보입니다.

**Comment by flaxinger** [comment]
1명 이상의 승인이 필요하도록 설정 변경하였습니다. 불편 끼쳐드려 죄송합니다.
